rpm/libmediaart.spec

```diff
@@ -1,8 +1,8 @@
 Name:       libmediaart
 Summary:    Library for handling media art
-Version:    1.9.4
+Version:    1.9.5
 Release:    1
-License:    GPLv2
+License:    LGPLv2
```

Comment:
```suggestion
License:    LGPLv2+
```

Reply:
Oh yes, that seems to be even more precise.

Reply:
Updated.